solution/build.sh

```diff
@@ -0,0 +1,180 @@
+{
+	cp -R -f operator/ ${ISO_ROOT}/operator
+	# sed "s/VERSION_FULL/${VERSION_FULL}/" zenko-version-cr.yaml > ${ISO_ROOT}/operator/zenko-version-cr.yaml
+}
+
+# function download_dependency_yamls()
+# {
+	# OPERATOR_DIR=${ISO_ROOT}/operator
+	# CRDS_DIR=${OPERATOR_DIR}/deploy/crds
+	# mkdir -p ${CRDS_DIR}
+
+	# CERT_VERSION=$(grep -oP '/cert-manager-controller:\K.+$' deps.txt)
+	# wget -O ${CRDS_DIR}/cert_manager_crd.yaml https://github.com/jetstack/cert-manager/releases/download/${CERT_VERSION}/cert-manager.yaml
+
+	# combine all zookeeper files into a bundle.yaml
+	# ZOOKEEPER_VERSION=$(grep /zookeeper-operator: deps.txt | awk -F ':' '{print $2}')
+	# ZOOKEEPER_CRD_FILE=${CRDS_DIR}/zookeeper_operator_crd.yaml
+	# wget -O ${BUILD_ROOT}/zookeeper-operator.tar.gz https://github.com/pravega/zookeeper-operator/archive/v${ZOOKEEPER_VERSION}.tar.gz
+	# mkdir ${BUILD_ROOT}/zookeeper-operator
+	# tar -C ${BUILD_ROOT}/zookeeper-operator --strip-components=1 -xf ${BUILD_ROOT}/zookeeper-operator.tar.gz
+	# cp ${BUILD_ROOT}/zookeeper-operator/deploy/crds/zookeeper_v1beta1_zookeepercluster_crd.yaml ${ZOOKEEPER_CRD_FILE}
+	# echo --- >> ${ZOOKEEPER_CRD_FILE}
+	# cat ${BUILD_ROOT}/zookeeper-operator/deploy/all_ns/rbac.yaml >> ${ZOOKEEPER_CRD_FILE}
+	# echo --- >> ${ZOOKEEPER_CRD_FILE}
+	# cat ${BUILD_ROOT}/zookeeper-operator/deploy/all_ns/operator.yaml >> ${ZOOKEEPER_CRD_FILE}
+# }
+
+
+# function gen_operator_yaml()
+# {
+	# sed "s/REPLACE_IMAGE/zenko-operator:${OPERATOR_TAG}/" operator.yaml > ${ISO_ROOT}/operator/operator.yaml
+# }
+
+# function gen_operator_config_yaml()
+# {
+	# CONFIG_PATH=${ISO_ROOT}/operator/operator-config.yaml
+	# echo apiVersion: solutions.metalk8s.scality.com/v1alpha1 > ${CONFIG_PATH}
+	# echo kind: OperatorConfig >> ${CONFIG_PATH}
+	# echo repositories: >> ${CONFIG_PATH}
+  	# echo "  ${VERSION_FULL}:" >> ${CONFIG_PATH}
+    	# echo "    - endpoint: metalk8s-registry/${PRODUCT_LOWERNAME}-${VERSION_FULL}" >> ${CONFIG_PATH}
+      	# echo "      images:" >> ${CONFIG_PATH}
+	# for dep in ${DEP_IMAGES[@]}; do
+		# SHORT_DEP=${dep##*/}
+        	# echo "        - ${SHORT_DEP}" >> ${CONFIG_PATH}
+	# done
+# }
+
+function copy_image()
+{
+	IMAGE_NAME=${1##*/}
+	FULL_PATH=${IMAGES_ROOT}/${IMAGE_NAME/:/\/}
+	mkdir -p ${FULL_PATH}
+	${SKOPEO} ${SKOPEO_OPTS} copy \
+		--format v2s2 --dest-compress \
+		--src-daemon-host ${DOCKER_SOCKET} \
+		docker-daemon:${1} \
+		dir:${FULL_PATH}
+}
+
+function dedupe()
+{
+	${HARDLINK} -c ${IMAGES_ROOT}
+}
+
+function build_registry_config()
+{
+	docker run \
+    	 	--name static-oci-registry \
+	 	--mount type=bind,source=${ISO_ROOT}/images,destination=/var/lib/images \
+		--mount type=bind,source=${ISO_ROOT},destination=/var/run \
+	 	--rm \
+    		docker.io/nicolast/static-container-registry:latest \
+		python3 /static-container-registry.py --omit-constants /var/lib/images
+	CONFIG_FILE=${ISO_ROOT}/static-container-registry.conf
+	mv ${CONFIG_FILE} ${ISO_ROOT}/registry-config.inc
```

Comment:
This seems to be required by solutions
```suggestion
	mv ${CONFIG_FILE} ${ISO_ROOT}/registry-config.inc.j2
```